easybuild/easyconfigs/l/LAST/LAST-869-intel-2017a.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'MakeCp'
```

Comment:
We have an existing easyconfig for `LAST` (for a more recent version) which uses `ConfigureMake` and does `make install prefix=%(installdir)s`.

But maybe that's not supported yet with this older version of `LAST`?

Reply:
@boegel: this version has only a simple `makefile`,  no configure provided.

Reply:
Which is exactly why we skip the `configure` step in the existing `LAST` easyconfig, see https://github.com/easybuilders/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/l/LAST/LAST-914-intel-2017b.eb .